ui/conductor/src/traits/electricDemand/electric.js

```diff
@@ -62,15 +62,15 @@ export function useElectricDemand(value) {
   const _v = computed(() => toValue(value));
 
   // return (v / d) or 0 if v is undefined
-  const div = (v, d) => v ? v / d : 0;
+  const div = (v, d) => v ? v / d : null;
 
   const realPower = computed(() => div(_v.value?.realPower, 1000));
   const realPowerUnit = computed(() => 'kW');
   const apparentPower = computed(() => div(_v.value?.apparentPower, 1000));
   const apparentPowerUnit = computed(() => 'kVA');
   const reactivePower = computed(() => div(_v.value?.reactivePower, 1000));
   const reactivePowerUnit = computed(() => 'kVAr');
-  const powerFactor = computed(() => _v.value?.powerFactor ?? 0);
+  const powerFactor = computed(() => _v.value?.powerFactor ? _v.value?.powerFactor : null);
```

Comment:
This isn't correct. If `powerFactor` is 0 then this returns null, whereas I expect you want to return null if powerFactor is null or undefined?